CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## unreleased
 
+* Updated all dependencies to the latest including eslint-plugin-jest 27.
```

Comment:
You don't have to mention the below module since it's dev-dependency.
Please write a changelog for updating the minimum node version.

```suggestion
* Updated all dependencies to the latest.
```

Reply:
OK